launcher/sdw_updater_gui/Updater.py

```diff
@@ -430,6 +430,65 @@ def _safely_start_vm(vm):
         sdlog.error(str(e))
 
 
+def should_launch_updater(interval):
+    sdlog.info("Starting SecureDrop Launcher")
```

Comment:
This results in a duplicate log line (since the same message is logged from `main`), is that intentional)

Reply:
nope, merge cruft. will delete.